ietf/templates/nomcom/view_feedback.html

```diff
@@ -54,6 +54,12 @@ <h3 class="mt-5" id="declined">Declined each nominated position</h3>
                     </tr>
                 {% endfor %}
             </tbody>
+	    <tr>
+		<td></td>
+		<td></td>
+		<td><b>Totals</b></td>
+		{% for fbtot in feedback_totals %}<td>{{ fbtot }}</td>{% endfor %}
+	    </tr>
```

Comment:
Should this be in a `tfoot`?

Reply:
yes, please make that change, consider `<th>` instead of the explicit `<b>`

Reply:
done.